dateparser/custom_lang_detect/fast_text.py

```diff
@@ -0,0 +1,31 @@
+import fasttext
+import os
+
+from dateparser.conf import apply_settings
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+model = "lid.176.ftz"
+_model_path = dir_path + '/data/' + model
+
+if os.path.exists(_model_path) == False:
+    import urllib.request
+    print("Downloading model", model)
+    url = "https://dl.fbaipublicfiles.com/fasttext/supervised-models/" + model
```

Comment:
Now that this is automatically downloaded, you should:

1. Add the file to the `.gitignore` file.
2. Remove the file from this PR